packages/block-editor/src/components/index.js

```diff
@@ -22,7 +22,7 @@ export { default as __experimentalGradientPickerControl } from './gradient-picke
 export { default as InnerBlocks } from './inner-blocks';
 export { default as InspectorAdvancedControls } from './inspector-advanced-controls';
 export { default as InspectorControls } from './inspector-controls';
-export { default as LinkControl } from './link-control';
```

Comment:
When was this component first introduced?

Reply:
this component is not being used, yet.

Reply:
we are integrating the `<LinkControl />` with the `<NavigationMenuItem />` on this PR: https://github.com/WordPress/gutenberg/pull/18062/

Reply:
It doesn't mean it was not used by plugins. Did we check at least in the plugins repository? Do you know when we introduce it?

Reply:
This branch was based on [try/unified-link-ui--experimental](https://github.com/WordPress/gutenberg/tree/try/unified-link-ui--experimental), and just merged with this one as well. The `<LinkControl />` component is totally new.

Reply:
Yep, as @retrofox mentions this PR wasn't merged to `master` but another branch that still has an open PR. `LinkControl` has never been merged to `master` so all should be fine.